openproblems/tasks/spatial_decomposition/datasets/__init__.py

```diff
@@ -1,2 +1,3 @@
-from .pancreas import pancreas_average
-from .pancreas import pancreas_cell
+# from .sc_to_sp import pancreas_alpha_0_1
```

Comment:
Why are these commented out?

Reply:
I commented them out cause they were failing in CI and wanted to first check whether, when commented out, CI would pass.